imagersite/imager_images/urls.py

```diff
@@ -1,13 +1,29 @@
 """Routes for user public images."""
 from django.conf.urls import url
 from imager_images.views import library_view
-from imager_images.views import photos_view
-from imager_images.views import albums_view
-
+from django.views.generic import ListView
+from imager_images.models import Photo
+from imager_images.models import Album
+from imager_images.views import AlbumsView
+from imager_images.views import AlbumAdd
+from imager_images.views import PhotoAdd
 
 urlpatterns = [
     url(r'^library/$', library_view, name='library'),
-    url(r'^photos/$', photos_view, name='photos'),
-    url(r'^albums/$', albums_view, name='albums'),
-    url(r'^albums/(?P<album_id>\d+)$', albums_view, name='album')
+    url(r'^photos/$', ListView.as_view(
+        template_name="imager_images/photos.html",
+        model=Photo,
+        context_object_name="photos",
+        queryset=Photo.objects.filter(published="PU")
+
+    ), name='photos'),
+    url(r'^albums/$', ListView.as_view(
```

Comment:
Cleaner if you put this in the views.py, particularly because you have this already.